scripts/travis/upload-to-registry.sh

```diff
@@ -0,0 +1,113 @@
+#!/bin/bash
+
+set -euxf -o pipefail
+
+DOCKERHUB_USERNAME=${DOCKERHUB_USERNAME:-"jaegertracingbot"}
+DOCKERHUB_TOKEN=${DOCKERHUB_TOKEN:-}
+QUAY_USERNAME=${QUAY_USERNAME:-"jaegertracingbot+github_workflows"}
```

Comment:
Looks like this isn't being take from the env var set as secret in the project. The right username for this is `jaegertracing+github_workflows`